### ISSUE TITLE
EPUB with missing structural markup not reported as WCAG violation

### ISSUE THREAD

**Issue by bitsgalore** [daisy/ace#212]
[This EPUB file](https://www.dbnl.org/tekst/_hol003190301_01/ebook/_hol003190301_01.epub) contains text that is structured like this:

    <div class="kop" id="toc-anchor36"><span class="cursief">Berichten.</span></div>
    <div class="onderkop" id="toc-anchor37">I. Drukfout.</div>
    <div class="plat">In mijn brief uit Parijs (zie vorig nommer) is een drukfout
    ingeslopen, die de lezers wel reeds zullen hebben gevonden. Waar gesproken wordt
    van het bekende schilderij van Mad: Vigée - Lebrun, voorstellend Marie Antoinette
    met haar kinderen, staat namelijk abusievelijk: Lodewijk XVIII, inplaats van
    <span class="vet">Lodewijk XVII.</span></div>

The above div elements represent a heading, a sub-heading and some main content. Looking at [WCAG](https://www.w3.org/TR/WCAG20/) 1.3.1:

> 1.3.1 Info and Relationships: Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. (Level A)

Success criteria are [here](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=131#qr-content-structure-separation-programmatic). 

The div elements don't contain landmark roles, and there is no native semantic markup. So shouldn't this raise a WCAG 2.0 A violation?

Note that it's entirely possible I'm simply misinterpreting WCAG myself (e.g. the remark that "Other techniques may also be sufficient if they meet the success criterion" allows rather a lot of space for interpretation).

**Comment by rdeltour** [comment]
@mattgarrish is right (of course 😄). Ace's shares the philosophy of the underlying aXe library (used to check Web accessibility violations), which is that we're trying to report zero false-positives. In other words, all the issues reported by Ace are _real_ accessibility issues (or else it's a bug 😅); when Ace is not sure or cannot tell, Ace says nothing and defers to a later manual checking.

Closing this issue as `invalid`, but feel free to continue the discussion if you have any comments or questions!

**Comment by bitsgalore** [comment]
@mattgarrish @rdeltour Thanks for the explanation, this is useful to know! (And yes looking at the long list of possible techniques mentioned in WCAG I totally understand testing for things like these would be a total nightmare!)
